### ISSUE TITLE
Discard inactive tabs when number of inactive tabs checks only current window.

### ISSUE THREAD

**Issue by AJolly** [rNeomy/auto-tab-discard#255]
Took me forever to figure out why this extension was improperly working for me.

It seems that Auto Tab Discard checks active or inactive tabs on a per window basis, rather than globally.

**Comment by rNeomy** [comment]
No, not really! It just checks every 10 minutes (by default) and discards tabs that have been active for more than 10 minutes.

**Comment by AJolly** [comment]
What's the best way to debug why a tab isn't getting closed?
 I do not see a tool tip show up.

One weird thing - the number of tabs listed as discardable on startup is wildly different than the number of tabs I have open. 
"'startup plug-in' 'number of tabs that can be discarded on startup' 101
I have 142 tabs currently open.

I Increased the max number of tabs to check at a single time.

**Comment by rNeomy** [comment]
Go to the options page and enable logging. Then open the background page of the extension.

Open "chrome://extensions" in a browser tab and turn the developer mode on. Click on the inspect button for this extension and switch to the console tab.

If you could find a way to reproduce the issue, please provide steps here.

**Comment by Chealer** [comment]
@AJolly : can you reproduce with a current version, and if so, which browser do you use?
